packages/server/src/api/controllers/row/index.ts

```diff
@@ -146,6 +146,23 @@ export async function search(ctx: any) {
   })
 }
 
+export async function searchView(ctx: Ctx<void, SearchResponse>) {
+  const { viewId } = ctx.params
+  const view = await sdk.views.get(viewId)
+  if (!view) {
+    ctx.throw(404)
+  }
+  const tableId = view.tableId
+
+  ctx.status = 200
+  ctx.body = await quotas.addQuery(
```

Comment:
Maybe should try the body assignment first. In case the query fails in future if we add a limit. I'm fairly certain the exception handling would result in a 400 for that type of exception, but I couldn't say for certain. Having the status after the successful query will play it safe.

Reply:
I just tested it, any error will ignore the 200: 
![image](https://github.com/Budibase/budibase/assets/15987277/8152e724-003e-4560-a698-8ddbda0fa5a7)